tests/fixtures/app_404_not_found/app.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask
+
+app = Flask(__name__)
+freeze_config = {'ignore_404_not_found': True}
```

Comment:
I'd rather use something like:
```suggestion
freeze_config = {
    'status_handlers': {
        '201': 'warn',
        '301': 'follow',
        '404': 'ignore',
        '418': 'my_module:custom_handler',
        '429': 'retry',
        '5xx': 'error',
    }
}
```